relay-server/src/services/buffer/mod.rs

```diff
@@ -138,8 +138,8 @@ impl PartitionedEnvelopeBuffer {
     /// The rationale of using this partitioning strategy is to reduce memory usage across buffers
     /// since each individual buffer will only take care of a subset of projects.
     pub fn buffer(&self, project_key_pair: ProjectKeyPair) -> &ObservableEnvelopeBuffer {
-        let mut hasher = FnvHasher::default();
-        project_key_pair.own_key.hash(&mut hasher);
+        let mut hasher = AHasher::default();
```

Comment:
As suggested in the docs you might want to create the hasher via `RandomState` with a fixed seed.